plugins/source/aws/docs/tables/aws_resiliencehub_app_versions.md

```diff
@@ -22,4 +22,5 @@ The following tables depend on aws_resiliencehub_app_versions:
 |_cq_parent_id|UUID|
 |account_id (PK)|String|
 |region (PK)|String|
+|app_arn (PK)|String|
```

Comment:
If we are adding `app_arn` doesn't that mean that `account_id` and `region` are duplicated?